packages/medusa-payment-paypal/src/core/types/payment.ts

```diff
@@ -0,0 +1,54 @@
+import { Links, MoneyAmount, PaymentInstruction } from "./common"
```

Comment:
todo(this-file): If you choose to incorporate my suggestions to the SDK method names, those changes should be reflected in this file.